### PR TITLE
add explanatory text

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

File: frontend/components/site/SiteSettings/index.jsx [hunk 1]
```diff
@@ -59,6 +59,11 @@ class SiteSettings extends React.Component {
 
     return (
       <div>
+        <p>
+          See our doucmentation site for more about
+          <a href="https://federalist-docs.18f.gov/pages/using-federalist/#managing-site-settings">these settings</a> or
```
Comment (el-mapache) [inline frontend/components/site/SiteSettings/index.jsx]: Do we want these to open in another tab? Most of our external links on federalist currently do so. I think we should also provide a `title` attribute for these.

Comment (wslack) [reply frontend/components/site/SiteSettings/index.jsx]: Good point

Comment (wslack) [reply frontend/components/site/SiteSettings/index.jsx]: I should also have target="_blank" rel="noopener noreferrer" in the link right?

Comment (el-mapache) [reply frontend/components/site/SiteSettings/index.jsx]: yes! You'll probably have to put the properties on a separate line to stay within our linter's line length requirements

Comment (jseppi) [reply frontend/components/site/SiteSettings/index.jsx]: yes

**Comment by wslack** [comment]
Oh thanks @el-mapache - was on my list to do.

**Comment by wslack** [comment]
Now I just need a review!

File: frontend/components/site/SiteSettings/index.jsx [hunk 2]
```diff
@@ -59,6 +59,11 @@ class SiteSettings extends React.Component {
...
```
Comment (jseppi) [inline frontend/components/site/SiteSettings/index.jsx]: did you still want these links to open in a new window?

**Comment by wslack** [comment]
Oh right, I have more to do here. One moment.

**Pull request merged**